app/forms/candidate_interface/gcse_institution_country_form.rb

```diff
@@ -0,0 +1,26 @@
+module CandidateInterface
+  class GcseInstitutionCountryForm
+    include ActiveModel::Model
+
+    attr_accessor :institution_country
+
+    validates :institution_country, presence: true
+
+    validates :institution_country,
+              inclusion: { in: COUNTRIES_BY_NAME }
```

Comment:
Should this be being saved as a country code i.e. 'US' rather than 'United States'.

Reply:
Hmm, I think that's a good idea, could you update to that?

Reply:
Have done 👍